### PR TITLE
Add basic working version of "Add Book" & "About Book" pages.

### PR DESCRIPTION
**Pull request by archthegit**
* built a basic working version of book page but milligram isnt integrated and the UI is bad.

### CODE REVIEW

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 1]
```diff
@@ -395,4 +370,33 @@ private double getBookRating(String bookId) {
...
+    bookEntity.setProperty("avgRating", book.getAvgRating());
+    datastore.put(bookEntity);
+  }
+
+  /**
+  * Returns the Book identified by the id, or
+  * null if no matching Book was found.
+  */
+  public Book getBook(String id) {
+    Query query = new Query("Book")
```
Comment (SuyashLakhotia) [inline src/main/java/com/google/codeu/data/Datastore.java]: I think using KeyFactory might be more efficient than this query? I would imagine running this filter would continue to scan the DB even after a result is found, which will almost never be useful.

File: src/main/webapp/addbook.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Add a New Book</title>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="/css/main.css">
+    <!--    <script src="/js/book-page-loader.js"></script>-->
```
Comment (SuyashLakhotia) [inline src/main/webapp/addbook.html]: Is it safe to delete these commented out lines instead?

File: src/main/webapp/js/book-page-loader.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
+
```
Comment (SuyashLakhotia) [inline src/main/webapp/js/book-page-loader.js]: Delete these empty lines?

File: src/main/webapp/js/user-page-loader.js [hunk 4]
```diff
@@ -16,6 +16,7 @@
 
 // Get ?user=XYZ parameter value
 const urlParams = new URLSearchParams(window.location.search);
+console.log(urlParams);
```
Comment (SuyashLakhotia) [inline src/main/webapp/js/user-page-loader.js]: Probably want to delete this debug statement before merging as well.

**Comment by SuyashLakhotia** [comment]
Also, take a look at the comments @karenfrilya97 left on PR #33 if there are relevant changes you can make here too.

**Comment by archthegit** [comment]
> Looks good! There are 2 comments from Suyash that haven't been implemented yet: deleting empty lines and removing debugging codes. But I don't think those will affect much so I'll just approve first for now.

**Pull request merged**